source/Node.js

```diff
@@ -91,7 +91,9 @@ function getPath ( node ) {
         if ( id = node.id ) {
             path += '#' + id;
         }
-        if ( className = node.className.trim() ) {
+
+
+        if (node.className && node.classList.trim && (className = node.className.trim()) ) {
```

Comment:
Agree with the first point. I agree with the second, but that's how it is in vanilla Squire.